### ISSUE TITLE
Incompatible with django-render-block==0.9 -> ModuleNotFoundError: No module named 'render_block'

### ISSUE THREAD

**Issue by bj00rn** [vintasoftware/django-templated-email#135]
`django-render-block` 0.9 was released 14 december

python 3.8.10

django-templated-email==2.3.0
Django==2.2.25

Getting error since recreating my virtualenv  and running pip install

```
...
    from templated_email import get_templated_mail
  File "/home/vagrant/.pyenv/versions/xxx_2474/lib/python3.8/site-packages/templated_email/__init__.py", line 6, in <module>
    from templated_email.backends.vanilla_django import TemplateBackend
  File "/home/vagrant/.pyenv/versions/xxx_2474/lib/python3.8/site-packages/templated_email/backends/vanilla_django.py", line 14, in <module>
    from render_block import render_block_to_string, BlockNotFound
ModuleNotFoundError: No module named 'render_block'
```

**Comment by mpibpc-mroose** [comment]
I've been hit by this, too.

**Comment by clokep** [comment]
I had an issue packaging it which should be resolved with django-render-block 0.9.1. I've also yanked 0.9 so otheres should not run into this issue.

I don't think there's anything to be done on django-templated-email side, so this can probably be closed.

**Comment by bj00rn** [comment]
closing as issue has been resolved in render_block
